src/features/swaps/SwapScreen.tsx

```diff
@@ -335,9 +335,10 @@ const SwapScreen = () => {
         Number(youPayTokenAmount),
         CurrencyType.networkToken,
       )
-      const amount = networkTokensToDc(networkTokens)?.floatBalance
-
-      return amount || 0
+      const rawBalance = networkTokensToDc(networkTokens)?.floatBalance
+      if (typeof rawBalance !== 'undefined') {
```

Comment:
If this supposed to check `typeof` for 'undefined' or that the `rawBalance` is `undefined`?

Would just `rawBalance !== undefined` work? I guess maybe they are the same thing

Reply:
I've never had direct comparisons to undefined work